code/modules/research/techweb/all_nodes.dm

```diff
@@ -304,6 +304,14 @@
 	design_ids = list("exodrone_console","exoscanner_console","exoscanner","exodrone_launcher")
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 2500)
 
+/datum/techweb_node/dex_robotics
+	id = "dex_robotics"
+	display_name = "Dexterous Robotics Research"
+	description = "The fine art of opposable thumbs."
+	prereq_ids = list("adv_engi", "adv_robotics", "biotech")
```

Comment:
if you have to have posibrains researched before you can print drones, why are you printing drones instead of posibrains? just material concerns? in my experience, when robotics actually bothers to make posibrains, there are usually more posibrains than there are ghosts willing to take them- let alone ghosts with ~1.67 days' worth of silicon playtime.

Reply:
not only that but putting a posi that's marked to be an engiborg then putting it in an unsynced borg is just going to make a better drone than a drone

Reply:
This may be rebalanced later

Reply:
> This may be rebalanced later, better to be underpowered than overpowered

is it, though? any PR that buffs drones is going to start another war in its comments section.

see: any PR that buffs PK borgs, or borgs in general

Reply:
it absoluteyl is

Reply:
and don't conflate people trying to make PK borgs into secborgs with changes to drones